index.html

```diff
@@ -33,5 +33,183 @@ <h2 class="social">LET'S CONNECT</h2>
+                </h2>
+                <div class="social-media">
+                    <span class="connect">
+                        LET'S CONNECT
+                    </span>
+                    <div class="social-media-icons">
+                        <ul>
+                            <li> <a href="#"><img src="./img/twitter-icon.svg" alt=""></a></li>
+                            <li><a href="#"><img src="./img/Linkedin-icon.svg" alt=""></a></li>
+                            <li><a href="#"><img src="./img/Matador-icon.svg" alt=""></a></li>
+                            <li><a href="#"><img src="./img/github_icon.svg" alt=""></a></li>
+                            <li><a href="#"><img src="./img/victory-icon.svg" alt=""></a></li>
+                        </ul>    
+                    </div>
+                </div>
+                <button class="action-button">Get My Resume</button>
+            </div>
+    
+            <div class="skillset">
+                <div class="languages">
+                        <div class="languages-heading">
+                            <span class="language-title">LANGUAGES</span>
+                            <button class="chevron-button">
+                                <img src="./img/chevron-down.png" alt="" class="chevron">
+                            </button>
+                        </div>
+                            <ul class="language-list">
+                                <li><img src="./img/JS-icon.png" alt="" class="language-icon">
+                                    <span class="language-name">JavaScript</span>    
+                                </li>
+                                
+                                <li><img src="./img/HTML5-icon.png" alt="" class="language-icon">
+                                    <span class="language-name">HTML</span>    
+                                </li>
+    
+                                <li><img src="./img/CSS-icon.png" alt="" class="language-icon">
+                                    <span class="language-name">CSS</span>    
+                                </li>
+    
+                                
+                            </ul>
+                            
+                        
+                    </div>
+                    <div class="frameworks">
+                        <span class="frameworks-title">Frameworks</span>
+                       <button class="chevron-button">
+                         <img src="./img/chevron-right.png" alt="" class="chevron">
+                        </button>
+                    </div>
+                    <div class="skills">
+                        <span class="skills-title">Skills</span>
+                        <button class="chevron-button">
+                            <img src="./img/chevron-right.png" alt="" class="chevron">
+                        </button>
+                    </div>
```

Comment:
- [ ] For this section kindly use the following structure:

```
<ul>
  <li>Languages
    <ul>
       <li>Javascript<li>
        .....
     </ul>
  </li>
  <li>Framworks</li>
  .....
</ul>
```

- [ ] Kindly remove the borders around the text in the about me section too to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205450958-ef61b83a-6a22-4756-9fb6-45c550d17498.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205450970-0b91b116-c005-4480-9bbe-fcd993051ad1.png)